client/cypress/integration/app.spec.ts

```diff
@@ -5,28 +5,16 @@ const page = new AppPage();
 describe('App', () => {
   beforeEach(() => page.navigateTo());
 
-  it('Should have the correct title', () => {
-    page.getAppTitle().should('contain', 'Handy Pantry');
+  it('Should have a working toolbar', () => {
+    cy.get('.toolbar-item').should('contain.text', 'My Pantry');
   });
 
-  it('The sidenav should open, navigate to "Products" and back to "Pantry"', () => {
-    // Before clicking on the button, the sidenav should be hidden
-    page.getSidenav()
-      .should('be.hidden');
-    page.getSidenavButton()
-      .should('be.visible');
-
-    page.getSidenavButton().click();
+  it('Should be able to go to the pages', () => {
```

Comment:
For some reason this `it` clause wasn't being run when it was nested inside the previous `it`. It wasn't clear why it was nested there in the first place, so I pulled it out and these tests are now running.